src/component/menu/__tests__/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
can use `overrides` in the root eslintrc to avoid this new file. No biggie, though

Reply:
of course, that will be better :+1: